### PR TITLE
2 packages from jobjo/popper at 0.1.1

### PR DESCRIPTION
**Pull request by jobjo**
This pull-request concerns:
-`popper.0.1.1`: Property-based testing at ease
-`ppx_deriving_popper.0.1.1`: A ppx deriving sample-functions for Popper

---
* Homepage: https://github.com/jobjo/popper
* Source repo: git+https://github.com/jobjo/popper.git
* Bug tracker: https://github.com/jobjo/popper/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/ppx_deriving_popper/ppx_deriving_popper.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+maintainer: ["Joel Bjornson <joel.bjornson@gmail.com>"]
+authors: ["Joel Bjornson <joel.bjornson@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/jobjo/popper"
+doc: "https://jobjo.github.io/popper/"
+bug-reports: "https://github.com/jobjo/popper/issues"
+depends: [
+  "ppxlib"
+  "ppx_deriving" {>= "5.0"}
+  "popper"
```
[the remark this answers is not in the document]
Comment (jobjo) [reply packages/ppx_deriving_popper/ppx_deriving_popper.0.1.1/opam]: Yep, correct. Thanks for the tip. Updated.

File: packages/popper/popper.0.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "Popper (after Karl) is an OCaml library for writing regular unit tests as well as property-based tests. The design is inspired by the Python library Hypothesis and supports built-in shrinking for counter-examples."
+maintainer: ["Joel Bjornson <joel.bjornson@gmail.com>"]
+authors: ["Joel Bjornson <joel.bjornson@gmail.com>"]
+license: "ISC"
+homepage: "https://github.com/jobjo/popper"
+doc: "https://jobjo.github.io/popper/"
+bug-reports: "https://github.com/jobjo/popper/issues"
+depends: [
+  "pringo"
+  "ocaml" {>= "4.09.1" & < "4.13"}
```
[the remark this answers is not in the document]
Comment (jobjo) [reply packages/popper/popper.0.1.1/opam]: Failed on CI for 4.13 (which I believe was unrelated). However, for the ppx, ppxlib has the same constraint why I added it here. Now removed so we'll see what happens when the build completes.

**Comment by jobjo** [comment]
@kit-ty-kate noticed you pushed a change with a lower bound on ppxlib. What happens when the OPAM file here is out of synch with the one from the git-tag? Shall I just mirror and update the tagged version on github?

**Comment by kit-ty-kate** [comment]
> What happens when the OPAM file here is out of synch with the one from the git-tag?

Nothing. Opam does not get any information from the opam file in the archive/git-tag (unless you're pinning this git-tag exactly instead of using opam-repository but that's asking for trouble).
Constraints will most likely change in the future so changing the tag everytime this happens would break everything all the time.

However, it is very encourage to keep the opam file on your `main` branch in sync as it will help merge future releases faster and the opam file is used by users who want to try the dev version (using `opam install .` in your repository, for instance)

**Comment by jobjo** [comment]
Thanks @kit-ty-kate

> Nothing. Opam does not get any information from the opam file in the archive/git-tag (unless you're pinning this git-tag exactly instead of using opam-repository but that's asking for trouble).
> Constraints will most likely change in the future so changing the tag everytime this happens would break everything all the time.

Ok, makes sense.

> However, it is very encourage to keep the opam file on your `main` branch in sync as it will help merge future releases faster and the opam file is used by users who want to try the dev version (using `opam install .` in your repository, for instance)

Sure.

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**